gettsim/transfers/kinderzuschl/kost_unterk.py

```diff
@@ -18,21 +18,21 @@ def kinderzuschlag_kosten_unterk_m(
     ----------
     wohnbedarf_eltern_anteil
         See :func:`wohnbedarf_eltern_anteil`.
-    kinderzuschlag_kaltmiete_m
-        See :func:`kinderzuschlag_kaltmiete_m`.
-    kinderzuschlag_heizkost_m
-        See :func:`kinderzuschlag_heizkost_m`.
+    kinderzuschl_kaltmiete_m
+        See :func:`kinderzuschl_kaltmiete_m`.
+    kinderzuschl_heizkost_m
+        See :func:`kinderzuschl_heizkost_m`.
 
     Returns
     -------
 
     """
     return wohnbedarf_eltern_anteil * (
-        kinderzuschlag_kaltmiete_m + kinderzuschlag_heizkost_m
+        kinderzuschl_kaltmiete_m + kinderzuschl_heizkost_m
     )
 
 
-def kinderzuschlag_kaltmiete_m(
+def kinderzuschl_kaltmiete_m(
     hh_id: IntSeries, bruttokaltmiete_m_hh: FloatSeries, tax_unit_share: FloatSeries
 ) -> FloatSeries:
     """Calculate costs of living without heating costs.
```

Comment:
Docstring seems very weird. (Aside)

Reply:
"Calculate share of monthly rent expenses of the household that is attributed to the tax unit of the individual" ?

I think this function should actually be on the tax unit level and be named `bruttokaltmiete_m_tu`. But nothing I would like to change in this PR.

Reply:
"Share of household's monthly rent attributed to the tax unit."

Sounds like another issue to be opened :wink: 

Thanks!